buf.gen.yaml

```diff
@@ -0,0 +1,6 @@
+version: v1
+managed:
+  enabled: false
+plugins:
+  - plugin: buf.build/protocolbuffers/java:v23.4
```

Comment:
To remove a dependency on the BSR as part of the build, you may wish to use https://github.com/google/protobuf-gradle-plugin and https://github.com/bufbuild/buf-gradle-plugin/ instead (which will allow you to configure code generation to work entirely locally).